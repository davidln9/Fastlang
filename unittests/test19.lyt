
int main() {

	int x = 1;

	if (x == 3) {
		println("x is 3");
	} else {
		println("x is not 3");
	}
}
