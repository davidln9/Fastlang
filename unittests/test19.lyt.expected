x is not 3
